### PR TITLE
Working on the activity selection modal

### PR DESCRIPTION
**Pull request by NielEche**
- Cleaned up the add activity page
- finished the view all activity page
- working on the activity selection modal now

### CODE REVIEW

File: public/frontend/css/splash.css [hunk 1]
```diff
@@ -72,12 +73,11 @@ p {
     color: #0B24FA;
     font-weight: bold;
 }
-
 .btn:focus,
 .btn:active {
```
Comment (Jeremiahiro) [inline public/frontend/css/splash.css:77]: Always resolve conflicts after a pull request

Comment (NielEche) [reply public/frontend/css/splash.css:77]: yeah , i understand

File: routes/web.php [hunk 2]
```diff
@@ -15,12 +15,16 @@
...
+    return view('activity.index');
 });
 
-Route::get('fulfilled', function () {
-    return view('homepage.fulfilled');
+Route::get('create', function () {
+    return view('activity.create');
+});
+
+Route::get('activitySelection', function () {
```
Comment (Jeremiahiro) [inline routes/web.php:26]: subsequently, you could do 
`Route::get('/activity/selection', function() {
return view('link.to.the.view');})->name('activity.selection);`

That way you could use it as 
`{{ route('activity.selection') }} or {{ url('activity/selection') }}`


Don't use camel case of underscore and things of such to name routes.

**Comment by Jeremiahiro** [comment]
See the reviews. Thanks

File: routes/web.php (hunk 2 above)
Comment (NielEche) [reply routes/web.php:26]: ohh , noted . Thanks

**Pull request merged**